pkgs/development/tools/database/prisma-engines/default.nix

```
@@ -27,7 +29,7 @@ rustPlatform.buildRustPackage rec {
 
   cargoSha256 = "sha256-CQFTO43/b0ymYtnAtvySrp8cOF7rRdzY8mvcYTiiUp0=";
 
-  nativeBuildInputs = [ pkg-config ];
+  nativeBuildInputs = [ pkg-config git cargo ];
```

Cargo is already in nativeBuildInputs since this is a Rust package :)

(what made you add it?)

```suggestion
  nativeBuildInputs = [ pkg-config git ];
```

---

> Cargo is already in `nativeBuildInputs` since this is a Rust package :)

:disappointed: Oh, man! Thanks for the heads up! I've removed that explicit dependency.